networkx/utils/backends.py

```diff
@@ -523,6 +526,10 @@ def __call__(self, /, *args, backend=None, **kwargs):
                         fallback_to_nx=self._fallback_to_nx,
                     )
                 # All graphs are backend graphs--no need to convert!
+                _logger.debug(
```

Comment:
It might be good to have an env variable, like `NETWORKX_BACKEND_DEBUG`, and then only display these logging messages when this env variable is `True`. 

Or use `_logger.info` instead of `_logger.debug` if you think this msg should be displayed to the user every time an algorithm is dispatched to a backend implementation. 

also, what would happen if a backend implementation calls another backend function? does that internally used function call also get logged?

Thank you :)

Reply:
we can also add things like debug, caching, `NETWORKX_AUTOMATIC_BACKENDS`, and other backend env variables to networkx config.. this might also make it easier to implement @eriknw’s idea of bundling all the backend environment variables and ignoring them.

[Anthony] adding log levels(`DEBUG`, `INFO`, `WARN`, `ERROR`, etc.) (ref. [docs](https://docs.python.org/3/howto/logging.html))